README.md

```diff
@@ -46,6 +46,26 @@ const sub = client.subscribe(
 )
 ```
 
+#### Resending historical data 
+
+```javascript
+const sub = await client.resend(
```

Comment:
Would it make sense for consistency to pass streamId in the options object similar to `subscribe` method? I.e.,
```js
client.resend({
    stream: 'streamId',
    resend: {
        last: 5
    },
    ...
)
```

Reply:
Updated